CHANGELOG.asciidoc

```diff
@@ -193,6 +193,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Added support for haproxy 1.7 and 1.8. {pull}6793[6793]
 - Add accumulated I/O stats to diskio in the line of `docker stats`. {pull}6701[6701]
 - Ignore virtual filesystem types by default in system module. {pull}6819[6819]
+- Add connections metricset to RabbitMQ module {pull}6548[6548]
```

Comment:
Was this line added on purpose? Was it missing in a previews PR? If yes, the PR number seems to be off.